sirepo/pkcli/admin.py

```diff
@@ -141,9 +135,85 @@ def move_user_sims(target_uid=""):
         shutil.move(lib_file, target)
 
 
+def _build_ops(ops, simulations, sim_type, examples):
+    n = set([x.models.simulation.name for x in examples[sim_type]])
+    for sim in simulations:
+        if sim.name not in n:
+            ops.delete.append((sim, sim_type))
+        elif _example_is_too_old(sim.simulation.lastModified):
+            ops.revert.append((sim.name, sim_type))
+            ops.delete.append((sim, sim_type))
+
+
 def _create_example(example):
-    simulation_db.save_new_example(example)
+    try:
+        simulation_db.save_new_example(example)
+    except Exception as e:
```

Comment:
Why do we want to ignore exceptions here? Also, in the pkdlog below add `stack={}`. You can get the stack by calling `pkdexc()`

Reply:
In the event of a corrupted data file. Not exactly sure on the particulars here and waiting for @moellep to get back from vacation to clarify.

Reply:
I think this maybe can be removed now. The point of this exercise is to make sure we don't have corrupt examples.

Reply:
> I think the corrupted data file is if a user has an existing example simulation that is corrupted. Not if we have an example that we are trying to create that is corrupted. We'd want to know and crash in that case.

I think that is the correct case and I think it also would make sense to take fail fast approach